interop-testing/src/main/java/io/grpc/testing/integration/TestServiceServer.java

```diff
@@ -93,7 +97,15 @@ void parseArgs(String[] args) {
       }
       String value = parts[1];
       if ("port".equals(key)) {
-        port = Integer.parseInt(value);
+        port.clear();
+        for (String v : Splitter.onPattern("\\s+|,").split(value)) {
```

Comment:
I would split by ',' only. White space in command line is subtle. You can `v.trim()` before `parseInt`

Reply:
I actually didn't mean to mix whitespace and comma delimit, but sure, it's good to make it strict and straightforward, also align with --help message.